notebooks/00_scratch_r.Rmd

```diff
@@ -0,0 +1,223 @@
+---
+title: "00_scratch_r"
+output:
+#  html_document: default
+  pdf_document: default
+#  md_document:
+#    variant: markdown_github
```

Comment:
Why did you use github markdown?

Reply:
Because there are multiple flavors of `markdown` and since we're relying heavily on `Github` I simply chose their version.